utils/subreddit.py

```diff
@@ -34,7 +34,7 @@ def get_subreddit_undone(submissions: list, subreddit):
         if submission.stickied:
             print_substep("This post was pinned by moderators. Skipping...")
             continue
-        if submission.num_comments <= int(settings.config["reddit"]["thread"]["min_comments"]):
+        if submission.num_comments < int(settings.config["reddit"]["thread"]["min_comments"]):
```

Comment:
Do you not understand how equality operators work..

Reply:
yes I do, sir

if you set `min_comments` to 5 do you expect 5 comments to be valid? I do